testonly/hammer/hammer.go

```diff
@@ -412,60 +440,60 @@ func (s *hammerState) String() string {
 	return fmt.Sprintf("%d: lastSMR.rev=%d ops: total=%d (%f ops/sec) invalid=%d errs=%v%s", s.cfg.MapID, latestRev, totalReqs, float64(totalReqs)/interval.Seconds(), totalInvalidReqs, totalErrs, details)
 }
 
-func (s *hammerState) chooseOp(prng *rand.Rand) MapEntrypointName {
-	return s.cfg.EPBias.choose(prng)
+func pickIntInRange(min, max int, prng *rand.Rand) int {
+	delta := 1 + max - min
+	return min + prng.Intn(delta)
 }
 
-func (s *hammerState) chooseInvalid(ep MapEntrypointName, prng *rand.Rand) bool {
-	return s.cfg.EPBias.invalid(ep, prng)
+func (w *mapWorker) chooseOp() MapEntrypointName {
```

Comment:
These one-line methods are a bit annoying, esp. given that they are used **once** and from within the same type.
Consider inlining them here or in a separate PR.

Reply:
Agree. I see your refactoring style uses a big hammer than mine. I kinda like it.